src/main/java/org/motechproject/prevac/service/impl/CapacityInfoServiceImpl.java

```diff
@@ -55,13 +54,13 @@
             numberOfDays = numberOfDays < 0 ? 0 : numberOfDays;
 
             for (Clinic clinic : clinics) {
-                int visitCount = (int) visitBookingDetailsDataService.countFindByClinicIdAndBookingPlannedDateRange(clinic.getId(), dateRange);
-                int primeVacCount = (int) visitBookingDetailsDataService.countFindByClinicIdVisitTypeAndBookingPlannedDateRange(clinic.getId(),
+                int visitCount = (int) visitBookingDetailsDataService.countFindByClinicIdAndPlannedVisitDateRange(clinic.getId(), dateRange);
+                int primeVacCount = (int) visitBookingDetailsDataService.countFindByClinicIdVisitTypeAndPlannedVisitDateRange(clinic.getId(),
                         VisitType.PRIME_VACCINATION_DAY, dateRange);
-                int screeningCount = (int) screeningDataService.countFindByClinicIdAndDateRangeAndStatus(clinic.getId(), dateRange, ScreeningStatus.ACTIVE);
-                int unscheduledCount = (int) unscheduledVisitDataService.countFindByClinicIdAndDateRange(clinic.getId(), dateRange);
+                int screeningCount = (int) visitBookingDetailsDataService.countFindByClinicIdVisitTypeAndActualVisitDateRange(clinic.getId(),
```

Comment:
Screening screen is using it's own entity to schedule screening visits and we should use it (like it was before this change). Using actual date make no sense, because this visit already took place (was in the past) and you cannot schedule new visits in the past, so this list will always be empty.